### ISSUE TITLE
Add config/spice install info to the README

### ISSUE THREAD

**Issue by Kelvinrr** [USGS-Astrogeology/SugarSpice#84]
Config logic was changed to allow for optional out-of-source install area. Also there is an env var for the Spice data root directory. The build docs should be updated to reflect this. 

basically:
 * if `SSPICE_DEBUG` is set, it uses the repo's config files, otherwise uses `$CONDA_PREFIX/etc/SugarSpice/db`
 * It uses the following env in order of high to low priority: $SPICEROOT, $ALESPICEROOT, $ISISDATA
   * basically prioritize a specialized one, if not set try to use the ALE and then ISIS spice dir. The kernel searching logic is agnostic to NAIF vs ISIS installs.
